estate/models/estate_property_type.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models, api
+
+
+class EstatePropertyType(models.Model):
+    _name = "estate.property.type"
+    _description = "Real Estate Property Type"
+    _order = "sequence, name"
+
+    name = fields.Char(required=True)
+    property_ids = fields.One2many("estate.property", "property_type_id", string="Properties")
+    sequence = fields.Integer("Sequence", default=1)
```

Comment:
As I suggested to you in previous reviews, Odoo always generates a string if it same as the technical name, then no need to add.